app/public.js

```diff
@@ -70,13 +72,50 @@
         }
       }
     });
+    setEventListeners();
   }
 
   /**
+   * Refresh overlay
+   */
+  function refreshImagesInfo() {
+    hideImagesInfo();
+    showImagesInfo(null);
+  }
+
+  /**
+   * Set event listeners
+   */
+  function setEventListeners() {
+    let width = window.innerWidth;
+    let height = window.innerHeight;
+    let path = window.location.pathname;
+    let update;
+
+    interval = setInterval(function() {
```

Comment:
the idea is to not depend on a different method for clearing the interval. this couples those 2 methods together. in the case of calling showImagesInfo twice, it should be safe, including clearing existing overlays and existing intervals.

perhaps showImagesInfo should always call hideImagesInfo at the beginning to ensure a clean slate.